docs/canonicalk8s/snap/howto/refresh-external-certs.md

```diff
@@ -0,0 +1,93 @@
+# How to refresh externally managed Kubernetes certificates
+
+This guide walks you through refreshing external certificates for both control
+plane and worker nodes in your {{product}} cluster.
+
+## Prerequisites
+
+- A running {{product}} cluster
+
+```{note} To refresh certificates, your cluster must have been initially
+configured with external certificates during the bootstrap process. You can
+verify which certificates are externally managed by running `k8s certs-status`
+on a control plane node.
+```
+
+### Assemble certificates data
+
+To simplify the process and avoid complex CLI commands, the `refresh-certs`
+command accepts new node certificates via the `--external-certificates`
+argument with a YAML-formatted file. For a complete list of available
+certificate keys, see the [reference page].
+
+Refer to the {{ product }} [certificates] documentation to determine which
```

Comment:
suggestion: .. [cluster certificates and configuration directories] documentation to ...